packages/public/server/src/module/Common/src/Filter/Slugify.php

```diff
@@ -28,34 +28,51 @@ class Slugify implements FilterInterface
 {
     /**
      * @param string $text
-     * @return bool|mixed
+     * @return bool|string
      */
-    protected static function slugify($text)
+    protected static function slugify(string $text)
     {
-        $text = trim($text, ' ');
```

Comment:
Don't we want to have a trim of the text for articles with white spaces at the beginning or end of the name?

Reply:
Handled mostly be the rest of the regexp. It probably still adds a minus to the beginning / end if there's something like space, but tbh, I don't care much about that. Might even be the case that we clean the title up a little bit anyways.